xrpl/asyncio/ledger/main.py

```diff
@@ -54,13 +55,26 @@ async def get_fee(client: Client) -> str:
         client: the network client used to make network calls.
 
     Returns:
-        The minimum fee for transactions.
+        The minimum fee for transactions, in drops.
 
     Raises:
         XRPLRequestFailureException: if the rippled API call fails.
     """
-    response = await client.request_impl(Fee())
-    if response.is_successful():
-        return cast(str, response.result["drops"]["minimum_fee"])
+    response = await client.request_impl(ServerInfo())
+    if not response.is_successful():
+        raise XRPLRequestFailureException(response.result)
 
-    raise XRPLRequestFailureException(response.result)
+    server_info = response.result["info"]
+    if "validated_ledger" not in server_info:
+        raise XRPLRequestFailureException(response.result)
```

Comment:
Should these two conditions have different error messages? (Not successful vs `validated_ledger` missing)